File: activity/src/main/java/com.activity/broadcast/NetworkBroadCastReceiver.java
package com.activity.broadcast;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.support.v4.content.LocalBroadcastManager;

import com.activityutil.activity.Constants;

/**
 * Created by amit on 15/2/17.
 */

public class NetworkBroadCastReceiver extends BroadcastReceiver {

    @Override
    public void onReceive(Context context, Intent intent) {
        LocalBroadcastManager.getInstance(context)
                .sendBroadcast(new Intent(Constants.getActionBroadcastNetworkChanged()));
    }
}

File: activity/src/main/java/com/common/broadcast/NetworkBroadCastReceiver.kt
package com.common.broadcast

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.support.v4.content.LocalBroadcastManager

import com.common.Constants


/**
 * Created by amit on 15/2/17.
 */

class NetworkBroadCastReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        LocalBroadcastManager.getInstance(context)
                .sendBroadcast(Intent(Constants.getActionBroadcastNetworkChanged()))
    }
}
